plugin/core/settings.py

```diff
@@ -275,14 +276,20 @@ def update_client_config(external_config: ClientConfig, user_override_config: Di
     user_override_languages = read_language_configs(user_override_config)
     if not user_override_languages:
         user_override_languages = external_config.languages
+    user_override_settings = user_override_config.get("settings")
+    if isinstance(user_override_settings, dict):
```

Comment:
Not sure if it would make a difference but previous code has fallen back onto `external_config.settings` when Dict was false'y.

Reply:
Looking at it again, I guess it's equivalent now since `external_config.settings` is part of the dotted Dict.